plugins/classifier.cc

```diff
@@ -1076,6 +1076,11 @@ apply(const FunctionApplier & applier,
 
     std::tie(dense, fset, ts) = getFeatureSet(context, false /* attempt to optimize */);
 
+    if (fset->features.empty()) {
+        throw ML::Exception("The specified features couldn't be found in the "
```

Comment:
Peux-tu changer pour un truc comme: Must provide at least one non-null feature column to the classifier.

parce que tu peux en passer que le classifiieur connait pas et ça va pas planter. toi ton problème c'est qu'il n'y en a aucune.